workflows/virtual-screening-blueprint/scripts/run_diffdock.py

```diff
@@ -92,11 +95,17 @@ def build_result_field(v, t):
 
         time_start = time.time()
         response = client.infer(MODEL_NAME, inputs, request_id=str(uuid.uuid1()), outputs=outputs, timeout=TRITON_TIMEOUT)
-        logger.debug(f"Inference time: {time.time() - time_start} seconds")
+        print(f"Inference time: {time.time() - time_start} seconds")
 
         result = {v[2]: build_result_field(response.as_numpy(v[0]), v[1]) for v in MODEL_OUTPUT}
-        result['protein'] = body.protein
-        result['ligand'] = body.ligand
-        for i, v in enumerate(result):
-        	with open(f"{sys.argv[1].split('.')[0]}-{sys.argv[2].split('.')[0]}-{i}.json", 'w') as fh:
-        		fh.write(v)
+        if len(smiles_lines)>1:
+            for i, rs in enumerate(result['status']):
+                if rs=='success':
+                    for j, v in enumerate(result['ligand_positions'][i]):
+                        with open(f"{sys.argv[1].split('.')[0]}-{sys.argv[2].split('.')[0]}-{i}-{j}-score{result['position_confidence'][i][j]}.json", 'w') as fh:
```

Comment:
what drove this new difference? looks different than the above delete

Reply:
the if statement is to test if the output is a list, if so, we will loop through the list, otherwise, we can directly parse out the ligand_positions.